contracts/VetoGuard.sol

```diff
@@ -173,6 +173,8 @@ contract VetoGuard is
             ),
             "Transaction has been vetoed"
         );
+
+        require(!vetoERC20Voting.isFrozen(), "DAO is frozen");
```

Comment:
Ah werd splitting this out as a separate require statement is good

Reply:
Yes agreed! We should provide a user with specific error handling.